site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-mac.rst

```diff
@@ -14,34 +14,84 @@ It is advised to setup your Nextcloud devices on LAN (if available) for the best
-
 The smoothest experience will be using direct account integration with your Mac.  First head into the top-righthand menu of your Nextcloud's WebUI and click "Apps," then search for and install the Calendar and/or Contacts Apps.  The steps below are adapted from the `Official Nextcloud guide <https://docs.nextcloud.com/server/24/user_manual/en/groupware/sync_osx.html>`_ and updated for the latest MacOS (Ventura).  Make sure you have first set up :ref:`LAN access<lan-mac>`.
 
-1. Open the "System Settings" and select "Internet Accounts," then click "Add Account."
+.. tabs::
+  
+  .. group-tab:: Ventura
+
+    1. Open the "System Settings" and select "Internet Accounts," click "Add Account." and then select "Add Other Account".
+    
+      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step1.png
+        :width: 50%
+        :alt: macOS add account
+      
+      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step1.2.png
+        :width: 50%
+        :alt: macOS add account
+
+    
+    2. Select CalDAV for calendar setup or CardDAV for contacts setup. If you want to do both, you will need to return to this step after finishing the setup of the first.
+      
+      .. note:: You will need to perform 2 individual setups, one for Calendar and one for Contacts.
+      
+      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step2.png
+        :width: 50%
+        :alt: macOS select account
+    
+    3. Select "Advanced" from the "Account Type" dropdown menu and fill in the following fields:
+
+      .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step3.png
+        :width: 50%
+        :alt: macOS setup account
+      
+      - Username - The default user is "admin," but this is your user within Nextcloud, so be sure it is the correct user if you have more than one
+
+      - Password - In your Nextcloud WebUI, visit the top-right-hand menu and select "Personal Settings" -> "Security." At the bottom, under Devices & Sessions, create a new app password with a name of your choice, such as "MacCalDAV." Then, copy the resulting password into your Mac's account configuration. 
+      
+        .. figure:: /_static/images/services/nextcloud/native-nextcloud-macos-step4.png
+          :width: 50%
+          :alt: nextcloud app password 
+
+      - Server Address - copy your LAN address from the "Interfaces" section of your Nextcloud service page then paste without `https://` suffix.
+      
+      - Server Path - For CalDav enter this path `/remote.php/dav/principals/users/embassy/`. You can find complete path in Nextcloud -> Calendar settings -> Copy iOS/macOS CalDav address. For setting up contacts/CardDav use this path `/remote.php/dav`.
```

Comment:
This cannot be right if your nextcloud user name is admin.  I think it's best to just remove the beginning of sentence and lean on the instructions to copy the full path.  Then the second sentence should read:

For Contacts (CardDAV), append `/remote.php/dav` after `.local` to your LAN interface address.